replication-commands/src/main/java/org/codice/ditto/replication/commands/ConfigEnableCommand.java

```diff
@@ -43,23 +42,22 @@
   )
   List<String> configNames;
 
-  @Reference ReplicatorConfigLoader replicatorConfigLoader;
+  @Reference ReplicatorConfigManager replicatorConfigManager;
 
   @Reference Replicator replicator;
 
   @Override
   protected Object executeWithSubject() throws Exception {
     for (String config : configNames) {
-      ReplicatorConfig replicatorConfig = replicatorConfigLoader.getConfig(config).orElse(null);
+      ReplicatorConfig replicatorConfig = replicatorConfigManager.getConfig(config).orElse(null);
```

Comment:
These the config here is just the config name. I think you changed the getConfig to only work with the ID so this will need to be changed as well.